daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -165,6 +167,13 @@ newtype JsonApiOptions = JsonApiOptions [String]
 newtype ScriptOptions = ScriptOptions [String]
 newtype SandboxClassic = SandboxClassic { unSandboxClassic :: Bool }
 
+data CodegenConfig = CodegenConfig
```

Comment:
Do we actually use this type anywhere? I can't find a use site.

Reply:
no, that's a left over from the previous implementation, good catch!

Reply:
That's a left-over, I'll remove it. I still need to have another cleanup pass over the PR.